checks.d/tokumx.py

```diff
@@ -424,7 +424,11 @@ def collect_metrics(self, instance, server, conn, db, tags):
                 db_tags = list(tags)
                 db_tags.append('db:%s' % dbname)
                 db = conn[dbname]
-                stats = db.command('dbstats')
+                try:
+                    stats = db.command('dbstats')
+                except:
```

Comment:
Yes you should import the `OperationFailure` class (see http://api.mongodb.com/python/current/api/pymongo/errors.html ) and then `except OperationFailure`